plugin/pulp/plugin/importer.py

```diff
@@ -0,0 +1,18 @@
+from pulp.platform.models import RepositoryImporter
+
+
+class RepositoryImporter(RepositoryImporter):
+    """
+    The base RepositoryImporter object
+    """
+
+    def sync(self):
+        """
+        Perform a sync
```

Comment:
It would be great to emphasize here that the plugin writer should definitely not try to implement their sync logic on this class. They should implement it somewhere else, and just call their workflow from here.